src/test/kotlin/org/jitsi/nlj/module_tests/ReceiverFactory.kt

```diff
@@ -45,10 +45,10 @@ class ReceiverFactory {
             val receiver = RtpReceiverImpl(
                 Random().nextLong().toString(),
                 rtcpSender,
-                null,
                 RtcpEventNotifier(),
                 executor,
-                backgroundExecutor
+                backgroundExecutor,
+                { 0L }
```

Comment:
this is the method to get the send bitrate, right?  i think this will break tcc generation in the receiver test harness.

EDIT: i realized it would only be in the 'periodic feedback' mode of tccs, which we don't use.  i'm wondering know if we should just discuss if there really is a need to have both modes in here or if we should just stick with one and get rid of the other code.